docker/package/scripts/tezos-node-start

```diff
@@ -6,41 +6,36 @@
 
 set -euo pipefail
 
-: "${BIN_DIR:="/usr/bin"}"
```

Comment:
AFAIU it means "if `BIN_DIR` is unset, set it to `"/usr/bin"`, otherwise just use the value assigned to it". Correct? If so, does it mean that `BIN_DIR` previously was configurable, but now you removed this feature because it's unused and is not useful?

Also if it's correct, then probably statement "because node data dir is hardcoded in the startup script" is not entirely valid because one could configure it by specifying `DATA_DIR` (which would be suffixed with `/node/data`).

Reply:
All packages install binaries to `/usr/bin`, so yes, having it configurable looks useless.

> Also if it's correct, then probably statement "because node data dir is hardcoded in the startup script" is not entirely valid

Well, yes, but I think that having `DATA_DIR` explicitly defined in the config file is less confusing